### PR TITLE
send_as_sovereign as versioned xcm

### LINKED ISSUE #643: Update send_as_sovereign to use versioned XCM structs

### PR DESCRIPTION
**Pull request by zqhxuyuan**
use VersionedXcm and VersionedMultiLocation

closes #643

### CODE REVIEW

**Review by xlc — APPROVED** [review]

File: xcm/src/lib.rs [hunk 1]
```diff
@@ -6,8 +6,10 @@
 use frame_support::{pallet_prelude::*, traits::EnsureOrigin};
 use frame_system::pallet_prelude::*;
 use sp_std::boxed::Box;
+use sp_std::convert::{TryFrom, TryInto};
 
 use xcm::latest::prelude::*;
+use xcm::{VersionedMultiLocation, VersionedXcm};
```
Comment (zjb0807) [inline xcm/src/lib.rs:12]: These dependencies can be merged together

Comment (zjb0807) [reply xcm/src/lib.rs:12]:
```
use sp_std::{boxed::Box, convert::{TryFrom, TryInto}};
use xcm::{latest::prelude::*, VersionedMultiLocation, VersionedXcm};
```

**Comment by zjb0807** [comment]
This can fix it
```
cargo fmt --all
```

**Comment by zqhxuyuan** [comment]
> This can fix CI
> 
> ```
> cargo fmt --all
> ```

done, maybe we need githooks done this for us automatically?

**Comment by xlc** [comment]
If you are using orml within Acala, the githook of Acala should do the auto format

https://github.com/AcalaNetwork/Acala/blob/master/Makefile#L119

**Review by zjb0807 — APPROVED** [review]

**Pull request merged**